Jenkinsfile.release.azure

```diff
@@ -0,0 +1,11 @@
+node {
+  stage('Helm install') {
+    print 'Deploying to Azure'
+    withKubeConfig([credentialsId: 'azukubeconfig001']) {
+      sh "helm repo add ffc-demo http://gitlab.ffc.aws-int.defra.cloud/helm/helm-charts/raw/master/"
+      sh "helm repo update"
+      sh "helm fetch --untar ffc-demo/ffc-demo-user-service"
+      sh "helm upgrade --install --recreate-pods --wait --atomic --namespace azu-ffc-demo -f ffc-demo-user-service/production-values.yaml ffc-demo-user-service-azure ./ffc-demo-user-service"
```

Comment:
I can't access the cluster to verify the deployment works, but the deployment does not appear to override username, hostname or password for the database, so I'm not sure the container will start.

The script deploys to the same namespace and helm chart name as the job that deploys/undeploys the whole application for the federation investigation. It might be better to isolate these instances of the services to keep the federation deployment consistent.

Reply:
This is something we need to discuss in the pipeline discussion. How do we pass those values through? In the build pipeline we're overriding them in a values file, whereas the Azure pipeline overrides using variables set in the pipeline config. The approach we use guides how this will be done, so I've left this as is for now. It might result in the broken deployment, but I don't think that's an issue at this stage.